# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/WebIdentityChild.cpp
# Commit: ccd96b99aea2
# Full Hash: ccd96b99aea28e8214e585b864d3fb2245bac142
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing failures complaining about fedcm-no-registered-idps.https.html.
#   
#   This reverts commit ba33457e0cbac24a8b7d963c45138c62e92a74ae.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4bfadfc33207 -r ccd96b99aea2 dom/credentialmanagement/identity/WebIdentityChild.cpp
--- a/dom/credentialmanagement/identity/WebIdentityChild.cpp	Fri Jun 06 21:07:41 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,23 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "mozilla/dom/WebIdentityChild.h"
-#include "mozilla/dom/WebIdentityHandler.h"
-
-namespace mozilla::dom {
-
-void WebIdentityChild::ActorDestroy(ActorDestroyReason why) {
-  if (mHandler) {
-    mHandler->ActorDestroyed();
-    mHandler = nullptr;
-  }
-}
-
-void WebIdentityChild::SetHandler(WebIdentityHandler* aHandler) {
-  mHandler = aHandler;
-}
-
-}  // namespace mozilla::dom